deepomatic/api/http_helper.py

```diff
@@ -55,6 +59,8 @@ def __init__(self, app_id=None, api_key=None, verify_ssl=None, host=None, versio
         if app_id is None or api_key is None:
             raise DeepomaticException("Please specify 'app_id' and 'api_key' either by passing those values to the client or by defining the DEEPOMATIC_APP_ID and DEEPOMATIC_API_KEY environment variables.")
 
+        self.http_retryer = http_retryer or HTTPRetryer()
```

Comment:
maybe move default to class, instead of fancy default parameters on the constructor, like urllib3: HttpRetry.DEFAULT